sdk/src/constants.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Number of milliseconds between attempts to get a response from an embedded frame
+ */
+export const connectInterval = 500;
+
+/**
+ * How many times should we try to get an init response from an embedded frame
+ */
+export const connectMaxAttempts = 20;
+
+/**
+ * Default height attribute for iframes
+ */
+export const defaultFrameHeight = 300;
+
+/**
+ * Origin of the StackBlitz instance
+ */
+export const defaultOrigin = 'https://stackblitz.com';
+
+/**
+ * List of supported template names.
+ */
+export const projectTemplates = [
+  'angular-cli',
+  'create-react-app',
+  'html',
+  'javascript',
+  'node',
+  'polymer',
+  'typescript',
+  'vue',
+] as const;
```

Comment:
This array is used to warn users when they use an unsupported template type (with a `console.warn`), and is also used as the source of truth for the `ProjectTemplate` type:

```ts
type ProjectTemplate = typeof projectTemplates[number]; // 'angular-cli' | 'create-react-app' | 'html' | …
```